graph/vertex/TypeVertex.java

```diff
@@ -53,9 +53,9 @@
 
     void scope(String scope);
 
-    TypeAdjacency outs();
+    TypeAdjacency.Out outs();
 
-    TypeAdjacency ins();
+    TypeAdjacency.In ins();
```

Comment:
more strongly type the Adjacencies since different directions have different methods exposed